src/PSRule.Rules.Azure/rules/Azure.AKS.Rule.ps1

```diff
@@ -158,6 +158,21 @@ Rule 'Azure.AKS.AutoScaling' -Type 'Microsoft.ContainerService/managedClusters',
     }
 }
 
+# Synopsis: AKS clusters using Azure CNI should use large subnets to reduce IP exhaustion issues.
+Rule 'Azure.AKS.AzureCNI' -Type 'Microsoft.ContainerService/managedClusters', 'Microsoft.ContainerService/managedClusters/agentPools' -If { IsExport } -Tag @{ release = 'GA'; ruleSet = '2021_09'; } {
```

Comment:
Nice work on the implementation! I like the configuration option thought. The only suggestions are:

- Slightly change the name of the rule to be a bit more descriptive. Maybe `Azure.AKS.CNISubnetSize`.
- Capitalize the configuration option. Sorry this is not clear as there are different implementations. I'll update contributing.md. Capitalizing for consistency because of case-sensitivity requirements for Linux when configuring via environment variable. I'll need to implement a breaking change for v2 to update the existing configuration options that use mixed case but any new options should be capitialized.
- Add a entry in the rule defaults doc.

Reply:
Changed option to `Azure.AKS.CNISubnetSize`. Also capitalized configuration and added entrt to docs.